docs/pages/management/guides/joining-nodes-azure.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: Joining Nodes via Azure Managed Identity
+description: Use the Azure join method to add Nodes to your Teleport cluster on Azure
+---
+
+This guide will explain how to use the **Azure join method** to configure
+Teleport Nodes and Proxy Service instances to join your Teleport cluster without
+sharing any secrets when they are running in an Azure Virtual Machine.
+
+The Azure join method is available in Teleport 12.1+. It is available to any
+Teleport Node or Proxy running in an Azure Virtual Machine.
+
+<Details
+scope={["oss", "enterprise", "cloud"]}
+scopeOnly
+opened
```

Comment:
Let's change the title here. Token based joining isn't an "Azure joining method" - it's a generic join method that works on all infra.